apps/client-asset-sg/src/app/services/http.interceptor.ts

```
@@ -107,17 +107,7 @@ export class HttpInterceptor implements AngularHttpInterceptor, OnDestroy {
         break;
       default: {
         if (error.status < 500) {
-          this.store.dispatch(
```

so does this mean we are handling all the errors individually now? What happens to the errors that have previously been caught by this? Will they just fail silently?

---

Not all errors, just 4XX, which aren't meant to be handled globally anyways, as they indicate that the request itself did something wrong.